src/main/java/it/gov/pagopa/payhub/activities/service/classifications/trclassifiers/TransferClassifier.java

```diff
@@ -42,17 +42,27 @@ default Long getAmountCents(Transfer transferDTO) {
 	 * @param paymentsReportingDTO the payment reporting data.
 	 * @return the amount in cents, or {@code null} if the amount is not available.
 	 */
-	default Long getAmountCents(PaymentsReporting paymentsReportingDTO) {
+	default Long getAmountPaidCentsFromPaymentsReporting(PaymentsReporting paymentsReportingDTO) {
 		return Optional.ofNullable(paymentsReportingDTO).map(PaymentsReporting::getAmountPaidCents).orElse(0L);
 	}
 
+	/**
+	 * Extracts the total amount in cents from the payment reporting data.
+	 *
+	 * @param paymentsReportingDTO the payment reporting data.
+	 * @return the amount in cents, or {@code null} if the amount is not available.
+	 */
+	default Long getTotalAmountCentsFromPaymentsReporting(PaymentsReporting paymentsReportingDTO) {
+		return Optional.ofNullable(paymentsReportingDTO).map(PaymentsReporting::getTotalAmountCents).orElse(0L);
+	}
+
 	/**
 	 * Extracts the amount in cents from the treasury data.
 	 *
 	 * @param treasury the treasury data.
 	 * @return the amount in cents, or {@code null} if the amount is not available.
 	 */
-	default Long getAmountCents(Treasury treasury) {
+	default Long getBillAmountCentsFromTreasury(Treasury treasury) {
```

Comment:
why not the following?
```suggestion
	default Long getIufAmountCents(Treasury treasury) {
```